Gemfile

```diff
@@ -33,6 +33,7 @@ gem 'cancancan', '~> 2.0.0'
 gem 'github_webhook', '~> 1.1'
 gem 'clockwork'
 gem 'sidekiq'
+gem 'business_time'
```

Comment:
![giphy](https://user-images.githubusercontent.com/9951907/32486841-c2b756ec-c39f-11e7-9668-e8aef01d57e1.gif)

Reply:
i kinda wanted to spam this giphy anywhere it says `business_time`

Reply:
Why do you think I picked this gem?